Sources/PackageModel/Target.swift

```diff
@@ -517,14 +517,14 @@ public final class BinaryTarget: Target {
 
     public enum Kind: String, RawRepresentable, Codable, CaseIterable {
         case xcframework
-        // TODO: add new types
-        //case swiftLibraryArchive
-        //case swiftExecutableArchive
+        case toolsArchive
 
         public var fileExtension: String {
             switch self {
             case .xcframework:
                 return "xcframework"
+            case .toolsArchive:
+                return "toar"
```

Comment:
Not sure about this, but it's just name bikeshedding.

Reply:
yes, its terrible enough to be funny which is why I went with it. hoping the community would help us finalize that aspect.

Reply:
Ha!  Maybe ".this-is-something-we-have-to-change-before-shipping".  I can think of more than one example of placeholder UI that was intentionally made clunky so it would obviously have to be changed before shipping, and then not changed.

Reply:
came up with something slightly better imo